### ISSUE TITLE
Add external recommender integration tests

### ISSUE THREAD

**Issue by jcklie** [inception-project/inception#1251]
**Is your feature request related to a problem? Please describe.**
We now have several external recommender. We do not test them automaticaly against INCEpTION right now, which might lead to breaking them without us knowing.

**Describe the solution you'd like**
It would be nice to have integration tests for them so that we know we did not break them. I am not sure yet whether these tests should be inside INCEpTION or in a seperate repository.

**Comment by reckart** [comment]
Does anything speak against having them in same repo as the respective external recommender - i.e. they are executed as part of the build process of the external recommender?

**Comment by jcklie** [comment]
Yes, that is ok. We right now dont build external recommender though and would need Jenkins dependency magic to always build them when INCEpTION changes. That would maybe slow our build even more.

**Comment by reckart** [comment]
We could limit those build to a specific node to limit the impact of such builds.
